### PR TITLE
Adding public ipfs / private bacalhau instructions

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

**Comment by wesfloyd** [comment]
@fakela and @lukemarsden can I get your help to review the new instructions before I publish to main?

**Comment by wesfloyd** [comment]
Thanks!

File: docs/next-steps/private-cluster.md [hunk 1]
```diff
@@ -62,3 +64,37 @@ The exact command will be different on each computer and is outputted by the `ba
...
+```
+Monitor the output log for:
+11:16:03.827 | DBG pkg/transport/bprotocol/compute_handler.go:39 > ComputeHandler started on host QmWXAaSHbbP7mU4GrqDhkgUkX9EscfAHPMCHbrBSUi4A35
+
+
+On **all other nodes** execute the following:
+
+```
+export PEER_ADDR=/ip4/<public-ip>/tcp/1235/p2p/<above>
+export PEER_ADDR=/ip4/192.18.129.124/tcp/1235/p2p/QmWXAaSHbbP7mU4GrqDhkgUkX9EscfAHPMCHbrBSUi4A35
```
Comment (lukemarsden) [inline docs/next-steps/private-cluster.md:92]: This is an example, it's confusing like this

File: docs/next-steps/private-cluster.md [hunk 2]
```diff
@@ -62,3 +64,37 @@ The exact command will be different on each computer and is outputted by the `ba
...
 The command `export BACALHAU_IPFS_SWARM_ADDRESSES=...` sends jobs into the cluster from the command line client.
+
+## Public IPFS Network
+
+Instructions for connecting to the public IPFS network via private Bacalhau cluster:
+
+On all nodes, start ipfs:
+
+```
+export IPFS_CONNECT=$(ipfs id |grep tcp |grep 127.0.0.1 |sed s/4001/5001/|sed s/,//g |sed 's/"//g')
```
Comment (lukemarsden) [inline docs/next-steps/private-cluster.md:75]: This command doesn't start ipfs. You'll need to do that separately with ipfs init and ipfs daemon iirc... can you test these instructions?

Comment (fakela) [reply docs/next-steps/private-cluster.md:75]: To start ipfs we need to do 
`ipfs init` or `ipfs Daemon `

Then when do that command come in

File: docs/next-steps/private-cluster.md (hunk 1 above)
Comment (fakela) [reply docs/next-steps/private-cluster.md:92]: Let me make an update to this

**Pull request merged**